templates/citizen_project/_form.html.twig

```diff
@@ -103,6 +107,11 @@
             'placeholder': 'Votre calendrier prévisionnel et les principales étapes de votre projet',
         }
     }) }}
+    <small>
+        Ex. Nous vous proposons (i) de nous aider à identifier des professionnels susceptibles de proposer des stages
+        (ii) de faire connaître l'initiative auprès d'un maximum d'élèves, notamment via des associations (iii) de
```

Comment:
Where are the references `(i) ... (ii)` etc ? @guillaumedenhez @marineb

Reply:
ok my bad @cacahouwete can you adjust them like tabulations : 

Nous vous proposons :
(i)
(ii)
(iii)

Reply:
ok my bad @cacahouwete can you adjust them like tabulations :

Nous vous proposons :
(i)
(ii)
(iii)